schematic/utils/google_api_utils.py

```diff
@@ -8,13 +8,8 @@
 from google.auth.transport.requests import Request
 from google.oauth2 import service_account
 
-from schematic.utils.config_utils import load_yaml
-from definitions import ROOT_DIR, CONFIG_PATH, CREDS_PATH, TOKEN_PICKLE, CREDS_PATH, SERVICE_ACCT_CREDS
+from schematic import CONFIG
 
-config_data = load_yaml(CONFIG_PATH)
-
-# synapse ID of the 'credentials.json' file, which we need in order to establish communication with gAPIs/services
-SYN_CREDS = config_data["synapse"]["api_creds"]
```

Comment:
Because the config might not be loaded as soon as the module is imported, I moved any attempts to use the config within the relevant functions. This way, an attempt to use the config can come after the config is loaded.